invisible_cities/cities/beersheba.py

```diff
@@ -113,7 +117,10 @@ def deconvolve_signal(psf_fname       : str,
     diffusion     = np.asarray(diffusion              )
 
     psfs          = load_dst(psf_fname, 'PSF', 'PSFs')
-    deconvolution = deconvolve(n_iterations, iteration_tol, sample_width, bin_size, inter_method)
+    xygrid         = [np.arange(det_db[var].min() + bs/2, det_db[var].max() - bs/2 + np.finfo(np.float32).eps, bs)
```

Comment:
maybe the name is not the best, in principle this function should work even with ndim=3 and then it would be xyzgrid? how about `det_grid` (detector grid)?